pages/linux/lvm.md

```diff
@@ -0,0 +1,36 @@
+# lvm
+
+> Manage physical volumes, volume groups, and logical volumes using the Logical Volume Manager (LVM) interactivce shell.
+> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_logical_volumes>.
```

Comment:
Is it possible to have a more neutral man page link here? Perhaps <https://man7.org/linux/man-pages/man8/lvm.8.html> instead?

Reply:
@sbrl, see [this thread](https://github.com/tldr-pages/tldr/pull/5316#discussion_r584356393).

Reply:
Not sure I understand what you're getting at, @bl-ue.

Reply:
I was hoping to be able to link to link to a non-man page source of additional information. I find it  circular to have tldr point back to a man page. As a tldr user I appreciate being given an additional source of information that I might not have known about. 

I appreciate that this a larger discussion about what tldr More Information links should be and doesn't need to be resolved in the context of this PR. I'll change the link to be to man7.

Reply:
I see. I'm happy to have a another source of information here, but I'd really appreciate it if it were a non-versioned neutral web page. For example here, this is a link to the RHEL 8 docs. In the future, Oracle might decide that when RHEL 10 is out that they will delete or move the online docs for RHEL 8, rendering this link broken. Do you have any suggestions here?

Reply:
I agree that it's an ugly link and that it may break. I don't have anything prettier or more stable to suggest so I'm OK having it point back to the web page.

I thought I had seen in the guidelines a suggestion to use the More Information link to point to the project website (or something along those lines; I can't find that reference now). The project website (https://www.sourceware.org/lvm2/) isn't very informative and, since, I refer to the Red Hat guide a lot for infrequent procedures, I thought it would be a useful link. While it is version specific, RH is maintaining documentation for versions since version 4.